CHANGELOG.md

```diff
@@ -1,10 +1,12 @@
 CHANGELOG
 =========
 
-2.x.x
+2.11.8
 -----
 **CHANGES**
-Upgrade Python runtime used by Lambda functions in AWS Batch integration to python3.9.
+- Upgrade Python runtime used by Lambda functions in AWS Batch integration to python3.9.
+- Removed possibility to change tags when performing a `pcluster update`. 
```

Comment:
I'd call this as a bug fix, and slightly rephrase to something like "Preventing cluster tags to be changed during an update because not supported"